dev-util/idea-ultimate/idea-ultimate-2016.1.2.145.972.3.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Because I am a lazy git.
EAPI5 hasn't been deprecated and there's nothing to gain from an EAPI change.